client/public/index.html

```diff
@@ -4,6 +4,13 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+
+    <!-- roboto font,
+          there's probably a more webpacky way to do this, but this gets us there quick
+          and seems its what folks on the web have resolved solves the problem easily -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">
```

Comment:
If we are going to be deploying this in a network which doesn't have access to the internet, then maybe we would want to keep the font files under version control?